torch/csrc/jit/runtime/argument_spec.h

```diff
@@ -61,6 +55,8 @@ struct ArgumentInfo {
   int device_ : 8; // NOTE: this needs to be signed because we use -1 to
```

Comment:
This comment is out of date now.

Reply:
I will update the comments correspondingly.